app/views/pages/api-reference/liquid/sanitization.liquid

```diff
@@ -0,0 +1,65 @@
+---
+converter: markdown
+slug: api-reference/liquid/sanitization
+searchable: true
+metadata:
+  title: Liquid - Data sanitization
+---
+
+When you display data, from user input or from external sources, it is important to sanitize output before displaying it on a website to avoid XSS attacks.
```

Comment:
```suggestion
When you display data, from user input or from external sources, it is important to sanitize output before displaying it on a website to avoid [XSS](https://en.wikipedia.org/wiki/Cross-site_scripting) attacks.
```